WordPress/Classes/ViewRelated/Stats/Today Widgets/Shared Views/WidgetUnconfiguredCell.swift

```diff
@@ -4,6 +4,16 @@ enum WidgetType {
     case today
     case allTime
     case thisWeek
+    case loadingFailed
+
+    var configureLabelFont: UIFont {
+        switch self {
+        case .loadingFailed:
+            return UIFont.systemFont(ofSize: UIFont.preferredFont(forTextStyle: .headline).pointSize)
```

Comment:
Do the font configs belong in `WPStyleGuide+Stats`?

Reply:
Nice catch. `WPStyleGuide+Stats` is for the app Stats, but there is a `WidgetStyles` specifically for the Widgets. So I moved the font definitions there.